test/destory.test.js

```diff
@@ -0,0 +1,26 @@
+var fs = require('fs');
+var path = require('path');
+
+var test = require('tape');
+var pixelmatch = require('pixelmatch');
+
+var hijack = require('./hijack');
+var byebyte = require('../index');
+
+var width = 400;
+var height = 198;
+
+test('[destroy] basic opts', function(assert) {
+    hijack.hijack();
+    var fileBuffer = fs.readFileSync(path.join(__dirname, './logo.jpg'));
+    var expected = fs.readFileSync(path.join(__dirname, 'fixtures', 'destroy-basic-opts.jpg'));
+
+    var out = byebyte.destroy(fileBuffer, {min:.3, max:.5});
+
+    var diff = pixelmatch(expected, out, null, width, height, {threshold: 0.1});
```

Comment:
I think we should avoid using `pixelmatch` here and instead just do a direct comparison between the returned Buffer and the expected Buffer with [Buffer.prototype.equals](https://nodejs.org/dist/latest-v6.x/docs/api/buffer.html#buffer_buf_equals_otherbuffer), because byebyte can be used for any sort of file, and `pixelmatch` restricts us to testing against images.